test/spec/ol/renderer/canvas/replay.test.js

```diff
@@ -140,21 +141,33 @@ describe('ol.render.canvas.ReplayGroup', function() {
       expect(beginPathCount).to.be(3);
     });
 
-    it('applies the pixelRatio to the linedash array', function() {
-      var lineDash, lineDashCount = 0;
+    it('applies the pixelRatio to the linedash array and offset', function() {
+      var lineDash, lineDashCount = 0,
+          lineDashOffset, lineDashOffsetCount = 0;
 
       context.setLineDash = function(lineDash_) {
         lineDashCount++;
         lineDash = lineDash_.slice();
       };
 
+      Object.defineProperty(context, 'lineDashOffset', {
```

Comment:
Ah, thanks for the explanation. I had not thought about that! In this case your approach makes perfect sense.